priv/catalog/0B6DB2.yaml

```diff
@@ -48,14 +48,15 @@ remediation: |
     - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-storage-protect.html
     - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-1.11.5.5.4.4
 
+when: env.provider == "azure" || env.provider == "nutanix" || env.provider == "kvm"
```

Comment:
Maybe `env.provider in ["azure", "nutanix", "kvm"]` looks bettern ans shorter?
What do you think?

Reply:
yes, looks nice. I wasn't aware that I can use 'all' rhai expressions inside of the `when` clause
Will fix it soon, thanks